### PR TITLE
Add darkmode 2

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no dark mode available.

**What is the new behavior (if this is a feature change)?**
- The dark mode has been added and has been customized for the entire page.

**Other information**:
- The version has been updated to v1.5.0.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looking okay to me.

File: src/components/pages/BrandingPage/index.jsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ class BrandingPage extends React.Component {
                     >
                       <MDBIcon icon="download" className="pr-2" />
                       .svg
-                    </MDBBtn>
+                    </MDBBtn>jo
```
Comment (LucaAllmaier) [inline src/components/pages/BrandingPage/index.jsx]: typed in the wrong tab?

Comment (Aichnerc) [reply src/components/pages/BrandingPage/index.jsx]: Oh damn. This was just a test I used to see if the rigth component renders.

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

File: src/components/organisms/sections/Hero/index.jsx [hunk 2]
```diff
@@ -88,14 +91,18 @@ class HomePage extends React.Component {
 
   render() {
     const { darkMode } = this.props;
+    console.log(darkMode);
```
Comment (pinterid) [inline src/components/organisms/sections/Hero/index.jsx]: Add newline or remove console.log.

File: src/utilities/_base.scss [hunk 3]
```diff
@@ -112,6 +112,163 @@
...
+      }
+      .stepper.timeline li .step-content:before {
+        display: none;
+        border-left-width: 0;
+        border-right-width: 15px;
+        left: -15px;
+        right: auto;
+      }
+    }
+
```
Comment (pinterid) [inline src/utilities/_base.scss]: Is this newline intended?

**Review by pinterid — CHANGES_REQUESTED** [review]
Should be ok after a few changes.

**Review by pinterid — APPROVED** [review]
Looks good to merge.

File: src/components/organisms/sections/CallToAction/index.jsx [hunk 4]
```diff
@@ -23,7 +23,7 @@ import {
 //> Images
 // Rocket
 import rocket from "../../../../assets/content/icons/rocket.png";
-import planet1 from "../../../../assets/content/icons/planet1.jpg";
+import planet1 from "../../../../assets/content/icons/planet.png";
```
Comment (schettn) [inline src/components/organisms/sections/CallToAction/index.jsx]:
```suggestion
import planet from "../../../../assets/content/icons/planet.png";
```

File: src/components/organisms/sections/Hero/index.jsx [hunk 5]
```diff
@@ -29,6 +29,7 @@ import "./hero.scss";
 
 //> Settings
 import radarSettings from "./radarSettings.js";
```
Comment (schettn) [inline src/components/organisms/sections/Hero/index.jsx]: The file extensions here aren't required, are they?
```suggestion
import radarSettings from "./radarSettings";
```

File: src/components/organisms/sections/Hero/index.jsx [hunk 6]
```diff
@@ -29,6 +29,7 @@ import "./hero.scss";
...
+import radarSettingsDark from "./radarSettingsDark.js";
```
Comment (schettn) [inline src/components/organisms/sections/Hero/index.jsx]: The file extensions here aren't required, are they?
```suggestion
import radarSettingsDark from "./radarSettingsDark";
```

**Review by schettn — CHANGES_REQUESTED** [review]
Check before merging.

**Review by schettn — APPROVED** [review]
🚀

**Pull request merged**